AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/CascadeDeleteOperation.swift

```diff
@@ -295,6 +313,8 @@ public class CascadeDeleteOperation<M: Model>: AsynchronousOperation {
             // mutation?
             switch deleteInput {
             case .withIdentifier, .withIdentifierAndCondition:
+                let syncDeletionsCount = (queryAndDeleteResult.associatedModels.count + queryAndDeleteResult.deletedModels.count)
```

Comment:
this looks like code duplication, in this case, i believe you can just do this one above the `switch deleteInput` line, on line 301, or even higher up